scripts/linters/pylint_extensions.py

```diff
@@ -1782,6 +1778,65 @@ def visit_classdef(self, node):
             self.add_message('newline-below-class-docstring', node=node)
 
 
+class SingleLinePragmaChecker(checkers.BaseChecker):
+    """Custom pylint checker which checks if pylint pragma is used to disable
+    a rule for a single line only.
+    """
+
+    __implements__ = interfaces.IRawChecker
+
+    name = 'single-line-pragma'
+    priority = -1
+    msgs = {
+        'C0027': (
+            'Pylint pragmas should be used to disable a rule '
+            'for a single line only',
+            'single-line-pragma',
+            'Please use pylint pragmas to disable a rule for a single line only'
+        )
+    }
+
+    def process_module(self, node):
+        """Custom pylint checker which allows paramas to disable a rule for a
+        single line only.
+
+        Args:
+            node: astroid.scoped_nodes.Function. Node to access module content.
+        """
+        in_multi_line_comment = False
+        multi_line_indicator = b'"""'
+        file_content = read_from_node(node)
+
+        for line_num, line in enumerate(file_content):
+            line = file_content[line_num].lstrip()
+
+            # Single multi-line comment, ignore it.
+            if line.count(multi_line_indicator) == 2:
+                continue
+
+            # Flip multi-line boolean depending on whether or not we see
+            # the multi-line indicator. Possible for multiline comment to
+            # be somewhere other than the start of a line (e.g. func arg),
+            # so we can't look at start of or end of a line, which is why
+            # the case where two indicators in a single line is handled
+            # separately (i.e. one line comment with multi-line strings).
+            if multi_line_indicator in line:
+                in_multi_line_comment = not in_multi_line_comment
+
+            # Ignore anything inside a multi-line comment.
+            if in_multi_line_comment:
+                continue
+
+            # Ignore line that is enabling this check.
+            # Example: # pylint: enable:single-line-pragma
```

Comment:
Could you please change this Example to an example that is in codebase like `#pylint: enable:multi-line, single-line-pragma` and then explain it will enable this check. Once this check is enabled it will see pylint as the only comment in the line and hence raise an error/

Reply:
Done